apps/backoffice-v2/src/pages/Entity/components/Details/Details.tsx

```diff
@@ -25,9 +31,10 @@ export const Details: FunctionComponent<IDetailsProps> = ({ id, value, hideSepar
         workflowId={workflow?.id}
         id={id}
         valueId={value?.id}
-        documents={workflow?.context?.documents}
+        documents={documents ?? workflow?.context?.documents}
```

Comment:
I think we're better off making this component dumb. Accepting data from outside or falling back to specific data seems a bit confusing.

Reply:
Change of this could break places that im not aware of and could lead to bugs. I suggest keep it as it is right now and fix in future.